src/Aspire.ProjectTemplates/templates/aspire-starter/.template.config/template.json

```diff
@@ -254,6 +254,48 @@
         }
       ]
     },
+    "XUnitVersion": {
+      "type": "parameter",
+      "description": "The version of xUnit.net to use.",
+      "displayName": "xUnit.net version",
+      "datatype": "choice",
+      "isEnabled": "TestFramework == xUnit.net",
```

Comment:
this doesn't seem to be working
![image](https://github.com/user-attachments/assets/09cc5f9e-73e9-4464-a3f2-53b35e41f941)

Reply:
Maybe?
```suggestion
      "isEnabled": "(TestFramework == 'xUnit.net')",
```